vc-verifier/js/README.md

```diff
@@ -0,0 +1,22 @@
+# VC-Verifier
+
+## Features
+
+- Given any signed VC, the library verifies the VC for any tampering.
+
+
+## Installation
+
```

Comment:
lets add maybe the build and running test commands also

Reply:
done